src/ngraph_nnvm_ops.cc

```diff
@@ -190,12 +191,22 @@ void compute_backward(const mxnet::OpContext &ctx, std::shared_ptr<Graph> graph,
     }
   }
   check(req.size() == outputs.size());
-  auto results = get_tensors(
-      outputs, graph, false,
-      graph->bool_nodes_[mode][(int)(NodeReferences::kBackwardOutput)],
-      graph->scalar_nodes_[mode][(int)(NodeReferences::kBackwardOutput)], &req,
-      graph->is_reuse_mem);
-
+  TensorVector results;
+  if (is_train) {
```

Comment:
I don't think you need this branch, bprop should have the same number of results either way?